scripts/apis_scan.py

```diff
@@ -161,8 +163,19 @@ def enable_passive_scanner():
     zap.pscan.enable_all_scanners()
     zap.pscan.disable_scanners(disabledPassiveScan)
 
+def importurls(filepath):
+    """
+    Imports URLs (one per line) from the file with the given file system path.
+    This component is optional and therefore the API will only work if it is installed
+    """
+    return six.next(six.itervalues(zap._request(zap.base + 'exim/action/importUrls',
+                                                         {'filePath': filepath})))
 
 def get_APIs():
+    if urlScan:
```

Comment:
PEP8 recommends to use snake case in python:
https://peps.python.org/pep-0008/#method-names-and-instance-variables

But since it goes from YAML, we would need to create a model for it to "translate" YAML variables into python style